### ISSUE TITLE
Cannot provide a label for SplitViewPaneToggle

### ISSUE THREAD

**Issue by unindented** [winjs/winjs#1326]
For accessibility reasons, I'd like to provide a label for the button generated by `SplitViewPaneToggle`, but it doesn't seem to be possible.

**Comment by rigdern** [comment]
@unindented I tried providing an `aria-label` and it worked. Is there another property you are trying to set that isn't working? Can you provide repro steps?

Here's the code I used:

```html
<button
    aria-label="my label"
    class="win-splitviewpanetoggle"
    data-win-control="WinJS.UI.SplitViewPaneToggle"
    data-win-options="{ splitView: select('.splitView') }"
></button>
<script>
    WinJS.UI.processAll();
</script>
```

**Comment by unindented** [comment]
Ah, you are totally right. This seems to be an issue with `react-winjs` specifically. I was trying:

```js
<ReactWinJS.SplitViewPaneToggle
          aria-label="Menu"
          className="ApplicationHeader-toggle"
          paneOpened={paneOpened}
          onInvoked={onPaneToggle} />
```

And the `aria-label` property was ignored. I had assumed that it was a `winjs` problem.

I'll close and reopen in that project.
